app/src/main/assets/pdf/vendor/pdf.js/pdf_viewer.js

```diff
@@ -0,0 +1,5193 @@
+/* Copyright 2017 Mozilla Foundation
```

Comment:
Is there a minified version of this file? We could put the minified version on release build and this version on debug build so that we can read the code during development but when we ship the size is as small as it can be.

Also I think it's better to add a version suffix to this file so we know what PDF.js version we're using, and add an attribution to Mozilla in `license.xml`.